src/System.Text.Json/pkg/Microsoft.Bcl.Json.Sources.pkgproj

```diff
@@ -14,7 +14,7 @@
        the Common SR.cs into SourcePackageFiles. -->
   <Target Name="GetSourcesToPackage" BeforeTargets="ExpandProjectReferences">
     <ItemGroup>
-      <_ProjectsToBuild Include="../src/System.Text.Json.csproj" UndefineProperties="Configuration" />
+      <_ProjectsToBuild Include="../src/System.Text.Json.csproj" UndefineProperties="Configuration" AdditionalProperties="TargetGroup=netstandard"/>
```

Comment:
@joperezr, please review the packaging change :)

Reply:
Left a comment regarding the define constant that you declare for netstandard, since you probably don't want to require a constant to be defined in order for the source package to work. Other than that, this particular change on the pkgproj looks good to me.